src/modules/wallet/selectors.ts

```diff
@@ -0,0 +1,8 @@
+import { RootState } from 'modules/common/types'
+import { getData as getWallet } from 'decentraland-dapps/dist/modules/wallet/selectors'
+
+export function getManaBalance(state: RootState): number {
+  const wallet = getWallet(state)
+
+  return wallet ? wallet.networks.MATIC.mana : 0
+}
```

Comment:
Careful here, this method is misleading. There's no one mana balance, it always should depend on the network. Furthermore, decentraland dapps already has a `getNetworks` which can be accessed by network to get the mana balance (deprecating [this selector](https://github.com/decentraland/decentraland-dapps/blob/master/src/modules/wallet/selectors.ts#L36))
I think we can just use getNetworks where this selector is used and pass Netowrk.MATIC if necessary

Reply:
I took this section of code from [here](https://github.com/decentraland/builder/blob/c5c012568d9cdb5d437857597bd7f3d5b58651ef/src/components/Modals/PublishCollectionModal/PublishCollectionModal.tsx#L77).
Can't we have something like `getManaBalanceForNetwork(state:RootState, network: Network)`?  Because as far as I'm aware (I might be mistaken), the contract will live in the MATIC network. This also opens the window for the the question "should we should this modal at all if we're on another network that's not MATIC?" because we could disable the "Buy slots" button in that case.

Reply:
Created the `getManaBalanceForNetwork(state:RootState, network: Network)` method!